src/charts/SimplePowerBIReportEmbed/SimplePowerBIReportEmbed.js

```diff
@@ -50,19 +50,12 @@ const useStyles = makeStyles((theme) => ({
 }));
 
 function getErrorCode(labels, reports) {
-  let errorCode = labels.errors.unknown;
-  if (reports?.error?.status && reports?.error?.statusText) {
-    errorCode = `${reports?.error?.status} - ${reports?.error?.statusText}`;
-  }
-  return errorCode;
+  if (!reports?.error?.status && !reports?.error?.statusText) return labels.errors.unknown;
+  return `${reports?.error?.status ?? ''} ${reports?.error?.statusText ?? ''}`;
```

Comment:
This string syntax works without doubt but I am not very fan of it.

Reply:
I think this work :
`return (reports?.error?.status ?? '') + ' '  + (reports?.error?.statusText ?? ''};`

In any case, your first solution works anyway.